tests/oshiftroutetests/oshift_route_model_test.go

```diff
@@ -48,6 +48,7 @@ var ctrl *k8s.AviController
 
 var DefaultRouteName, DefaultNamespace, DefaultHostname, DefaultService string
 var DefaultModelName string
+var DefaultKey, DefaultValue string
```

Comment:
typically we name the variables with a lowercase if they are local and not to be exported as per Golang programming standards.